base/applications/utilman/registry.c

```diff
@@ -0,0 +1,201 @@
+/*
+ * PROJECT:         ReactOS Utility Manager (Accessibility)
+ * LICENSE:         GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:         Registry functions for Utility Manager settings management
+ * COPYRIGHT:       Copyright 2020 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+
+/* INCLUDES *******************************************************************/
+
+#include "precomp.h"
+
+/* GLOBALS ********************************************************************/
+
+UTILMAN_GLOBALS Globals;
+REGISTRY_DATA RegData;
+REGISTRY_SETTINGS Settings;
+
+/* DEFINES ********************************************************************/
+
+#define ACCESS_UTILMAN_KEY L"SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Accessibility\\Utility Manager";
+#define UTILMAN_KEY L"SOFTWARE\\Microsoft\\Utility Manager";
+#define OSK_KEY L"On-Screen Keyboard";
+#define MAGNIFIER_KEY L"Magnifier";
+
+/* FUNCTIONS ******************************************************************/
+
+/**
+ * @InitAppRegKey
+ *
+ * Initialize a key. The function may not necessarily create it but open the key
+ * if it already exists. The disposition pointed lpdwDisposition determines that.
+ * This is a function helper.
+ *
+ * @param[in]   hPredefinedKey
+ *     The predefined key (e.g. HKEY_CLASSES_ROOT).
+ *
+ * @param[in]   lpwszSubKey
+ *      The path to the sub key to be created.
+ *
+ * @param[out]   phKey
+ *      A pointer that receives a handle to the key given by the function.
+ *
+ * @param[out]   lpdwDisposition
+ *      A pointer that receives the disposition given by the function.
+ *
+ * @return
+ *      Returns TRUE if the function successfully created a key (or opened it),
+ *      FALSE otherwise for failure.
+ *
+ */
+BOOL InitAppRegKey(IN HKEY hPredefinedKey,
+                   IN LPCWSTR lpwszSubKey,
+                   OUT PHKEY phKey,
+                   OUT LPDWORD lpdwDisposition)
```

Comment:
Is this needed to know whether the utilman registry has been newly created or just opened?